blog/form.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
```

Comment:
Consider using crispy forms to enhance the form's appearance as per the task requirements. You can do this by adding `from crispy_forms.helper import FormHelper` and `from crispy_forms.layout import Submit` at the top of the file, and then configuring the `FormHelper` within the `CommentForm` class.